libs/s25main/ogl/ITexture.h

```diff
@@ -15,4 +16,5 @@ class ITexture
     virtual Position GetOrigin() const = 0;
     virtual Extent GetSize() const = 0;
     virtual void DrawFull(const Position& dstPos, unsigned color = 0xFFFFFFFFu) = 0;
+    virtual void Draw(Rect dstArea, Rect srcArea, unsigned color = 0xFFFFFFFFu) = 0;
```

Comment:
Please add a docstring here and what the method is supposed to do. `DrawFull(dstPos, color)` fully explains itself, but here it is unclear how the 2 areas are handled if they are not the same as the image size.